test/fetch-test-deps.sh

```diff
@@ -1,10 +1,46 @@
 #!/usr/bin/env bash
-
-set -e
+set -e  # stop the script when any command fails
```

Comment:
None of the other .sh scripts in here explain what `set -e` means.
```suggestion
set -e
```

Reply:
Is there any harm in a succinct explanation like this?

Reply:
I sought to condense the [existing explanation in `run-tests.sh`](https://github.com/gbdev/rgbds/blob/168950855d4d06eb4354e781b20da65a6833763b/test/run-tests.sh#L3C1-L3C55) and then standardize it across both files.